userprofile/static/userprofile/local_storage_external_services.js

```diff
@@ -19,9 +19,10 @@ $(function() {
 			if (key.indexOf("external_service_") != 0 || data === null)
 				continue;
 			console.log(data);
-			const li = $("<li>" + (data.title === undefined ? key : data.title) + "</li>");
-			li.addClass('list-group-item');
-			const btn = $('<button class="aplus-button--secondary aplus-button--sm pull-right"><i class="glyphicon glyphicon-remove"></i></button>');
+			const li = $("<li>" + (data.title === undefined ? key : data.title.split(' ')[0]) + "</li>");
+			li.addClass('list-group-item clearfix');
+      const btn = $('<button aria-label="Forget automatic login for ' + data.title.split(' ')[0] +
```

Comment:
> One line uses different indentation than others (tabs vs. spaces).
Yes, it is weird. my text editor does not show the indentation in the same way. Anyway, I will fix it.

Reply:
> my text editor does not show the indentation in the same way

Text editors typically render tabs instead of showing them as tabs (e.g. `^I`). `git diff` is usally good at showing this, as people usually don't use 8 characters in editor and I would expect they don't change the tab width in terminal.

To be clear, rest of the file uses tabs to indent while the new line uses space. The file could be added to the .editorconfig as an exception, if you like. If I remember right, there is quite many different identations in different files.